.github/workflows/test.yml

```
@@ -0,0 +1,209 @@
+            restore-keys: |
+              ${{ runner.os }}-pip-
+        - name: Install python dependencies
+          run: pip install -r requirements-dev.txt
+        - name: build documentation
+          run: make generate-doc
+        - name: Upload doc
+          uses: actions/upload-artifact@v2
+          with:
+            name: doc
+            path: doc/build/html
+            retention-days: 1
+
+  build-front:
+    name: Lint and build front end
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: "${{ env.NODE_VERSION }}"
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - name: Install front
+        run: make install-front
+      - name: Lint front end
+        run: make lint-front
+      - name: Build front
+        run: make build-front
+      - name: Upload assets for test
+        uses: actions/upload-artifact@v2
+        with:
+          name: assets
+          path: dist
+          retention-days: 1
+  test:
+    name: Install and test zds-site
+    needs: build-front
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        module: [
+          "zds.tutorialv2",
+          "zds.member zds.gallery zds.searchv2 zds.middlewares zds.pages",
+          "zds.forum zds.featured zds.mp zds.notification zds.utils"
+        ]
+    services:
+      elasticsearch:
+        image: elasticsearch:5.5.2
+        ports:
+          - "9200:9200"
+        env:
+          "http.host": "0.0.0.0"
+          "transport.host": "127.0.0.1"
+          "xpack.security.enabled": false
+          "ES_JAVA_OPTS": "-Xms512m -Xmx512m"
+        options: >-
+          -e="discovery.type=single-node"
+          --health-cmd="curl http://localhost:9200/_cluster/health"
+          --health-interval=10s
+          --health-timeout=5s
+          --health-retries=10
+      memcached:
+        image: memcached:1.6
+        ports:
+          - "11211:11211"
+    steps:
+      - name: Shutdown Ubuntu MySQL
+        run: sudo service mysql stop
+      - uses: getong/mariadb-action@v1.1
+        with:
+          character set server: 'utf8mb4'
+          collation server: 'utf8mb4_unicode_ci'
+          mariadb version: '10.4.10'
+          mysql database: 'ci_db_name'
+          mysql root password: 'ci_root_password'
+      - uses: actions/checkout@v2
+      - name: Download assets
+        uses: actions/download-artifact@v2
+        with:
+          name: assets
+          path: dist
+      - name: prints alls files
+        run: |
+          ls -rtlh dist
```

Pourquoi ? Est-ce un test qui est resté ?

Si en fait c'est important et à garder, typo.

```suggestion
      - name: Print all dist files
        run: |
          ls -rtlh dist
```